package.json

```diff
@@ -1,132 +1,132 @@
-	],
-	"homepage": "https://github.com/nhsevidence/global-nav",
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/nhsevidence/global-nav.git"
-	},
-	"license": "MIT",
-	"browserslist": [
-		"extends @nice-digital/browserslist-config"
-	],
-	"publishConfig": {
-		"registry": "https://registry.npmjs.org/",
-		"access": "public"
-	},
-	"engines": {
-		"node": ">= 6.9.0"
-	},
-	"devDependencies": {
-		"@babel/cli": "^7.2.3",
-		"@babel/core": "^7.2.2",
-		"@babel/plugin-syntax-dynamic-import": "^7.2.0",
-		"@babel/plugin-transform-member-expression-literals": "^7.2.0",
-		"@babel/plugin-transform-property-literals": "^7.2.0",
-		"@babel/plugin-transform-react-jsx": "^7.2.0",
-		"@babel/plugin-transform-reserved-words": "^7.2.0",
-		"@babel/plugin-transform-runtime": "^7.2.0",
-		"@babel/preset-env": "^7.2.3",
-		"@babel/preset-react": "^7.0.0",
-		"@hot-loader/react-dom": "^16.7.0",
-		"@nice-digital/eslint-config": "^1.0.3",
-		"autoprefixer": "^9.4.4",
-		"babel-eslint": "^10.0.1",
-		"babel-jest": "^24.0.0",
-		"babel-loader": "^8.0.4",
-		"clean-webpack-plugin": "^1.0.0",
-		"copy-webpack-plugin": "^5.0.1",
-		"copyfiles": "^2.1.0",
-		"core-js": "^2.6.1",
-		"cross-env": "^5.2.0",
-		"css-loader": "^2.1.0",
-		"enzyme": "^3.8.0",
-		"enzyme-adapter-react-16": "^1.8.0",
-		"enzyme-to-json": "^3.3.5",
-		"es3ify-loader": "^0.2.0",
-		"eslint": "^5.12.0",
-		"eslint-config-prettier": "^3.3.0",
-		"eslint-loader": "^2.1.1",
-		"eslint-plugin-jsx-a11y": "^6.1.2",
-		"eslint-plugin-prettier": "^3.0.1",
-		"eslint-plugin-react": "^7.12.3",
-		"html-webpack-exclude-assets-plugin": "0.0.7",
-		"html-webpack-plugin": "^3.2.0",
-		"identity-obj-proxy": "^3.0.0",
-		"jest": "^24.0.0",
-		"jest-teamcity-reporter": "^0.9.0",
-		"mockdate": "^2.0.3",
-		"nerv-devtools": "^1.3.9",
-		"node-sass": "^4.11.0",
-		"pixrem": "^5.0.0",
-		"postcss-loader": "^3.0.0",
-		"prettier": "1.15.3",
-		"promise-polyfill": "^8.1.0",
-		"react": "^16.7.0",
-		"react-dom": "^16.7.0",
-		"sass-loader": "^7.1.0",
-		"source-map-loader": "^0.2.4",
-		"style-loader": "^0.23.1",
-		"stylelint": "^9.10.1",
-		"stylelint-config-standard": "^18.2.0",
-		"stylelint-order": "^2.0.0",
-		"stylelint-scss": "^3.5.4",
-		"stylelint-webpack-plugin": "^0.10.5",
-		"uglifyjs-webpack-plugin": "^2.1.2",
-		"webpack": "^4.29.5",
-		"webpack-cli": "^3.2.3",
-		"webpack-dev-server": "^3.2.1",
-		"webpack-merge": "^4.2.1"
-	},
-	"dependencies": {
-		"@babel/runtime": "^7.3.4",
-		"@nice-digital/browserslist-config": "^1.0.0",
-		"@nice-digital/icons": "^2.1.1",
-		"@nice-digital/nds-core": "1.0.4-alpha.0",
-		"accessible-autocomplete": "^1.6.2",
-		"classnames": "^2.2.6",
-		"es5-polyfill": "0.0.6",
-		"js-cookie": "^2.2.0",
-		"nervjs": "^1.3.9",
-		"prop-types": "^15.6.2",
-		"react-hot-loader": "^4.7.1",
-		"unfetch": "^4.1.0"
-	},
-	"files": [
-		"lib",
-		"*.d.ts"
-	],
-	"directories": {
-		"lib": "lib"
-	}
+  "name": "@nice-digital/global-nav",
```

Comment:
I think we should use a prettierignore to ignore package.json (see https://github.com/nice-digital/nice-design-system/blob/v1/.prettierignore#L1)